lightning/src/chain/chainmonitor.rs

```diff
@@ -140,6 +141,29 @@ where C::Target: chain::Filter,
 		}
 	}
 
+	/// Gets the balances in the contained [`ChannelMonitor`]s which are claimable on-chain or
+	/// claims which are awaiting confirmation.
+	///
+	/// Includes the balances from each [`ChannelMonitor`] *except* those included in
+	/// `ignored_channels`, allowing you to filter for only channels which are not still open (and
```

Comment:
"allowing you to filter out channels which are still not open" ? What if the channel have received confirmation but not reached `CHAN_CONFIRM_DEPTH` ?

Reply:
Took your wording suggestion. If a channel is created but not yet confirmed it should still be in `ChannelManager::list_channels()`, so would be ignored here, which I think is the correct behavior.